include/base64.h

```diff
@@ -53,4 +59,55 @@ namespace lmms::base64
 
 } // namespace lmms::base64
 
-#endif
+namespace lmms::base64 {
+	constexpr inline std::array<char, 64> map =
+	{
+		'A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z',
+		'a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z',
+		'0','1','2','3','4','5','6','7','8','9',
+		'-','_'
+	};
+	const inline std::map<char, int> rmap {
```

Comment:
Why not use ``constexpr`` here as well? If anything, it will be consistent with ``map``.